packages/features/bookings/Booker/components/EventMeta.tsx

```diff
@@ -214,7 +219,10 @@ export const EventMeta = ({
                       container: () => "max-w-full",
                     }}
                     value={timezone}
-                    onChange={(tz) => setTimezone(tz.value)}
+                    onChange={({ value }) => {
+                      setTimezone(value);
+                      setBookerStoreTimezone(value);
```

Comment:
I don't see a reason to update bookerStore timezone when booker changes his preference.
It might make sense to do it if we update the 'cal.tz' query param as well based on it but I think we should think about it and introduce this enhacement later.

Right now we don't update the query param, so it might not be needed.

Reply:
Got the reason. It doesn't make sense for the booker page but it does for the booker component in Email Embed as there we read the bookerStore's timezone